rhobot.go

```diff
@@ -226,19 +231,43 @@ func healthcheckRunner(config *config.Config, healthcheckPath string, reportPath
 		_ = fhr.HandleReport(reader)
 	}
 
-	SMTPPortInt, _ := strconv.Atoi(config.SMTPPort)
-
 	// Email report
 	if emailListPath != "" {
-		ehr := report.EmailHandler{
-			SMTPHost:    config.SMTPHost,
-			SMTPPort:    SMTPPortInt,
-			SenderEmail: "-",
-			SenderName:  "-",
-			Subject:     "-",
-			Recipients:  []string{"-"},
-			HTML:        true,
+
+		SMTPPortInt, _ := strconv.Atoi(config.SMTPPort)
```

Comment:
Perhaps leave SMTPPort as a string until the last minute (i.e. in EmailHandler itself?)  That keeps the higher level functions a little cleaner.  Plus, it shouldn't really be an Int anyways, IMHO :)